main.cpp

```diff
@@ -265,6 +246,24 @@ struct DoubleType
     DoubleType& pow(const DoubleType&);
     DoubleType& pow(double);
 
+    DoubleType& apply(std::function<DoubleType& (double&)> DoubleFptr)
```

Comment:
Follow the coding standard in the course.
Variable names need to start with a lowercase letter.
- [ ] revise